config/jobs/kubernetes-sigs/cluster-api-boostrap-provider-kubeadm/cluster-api-bootstrap-provider-kubeadm-presubmits.yaml

```diff
@@ -0,0 +1,13 @@
+# sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm presubmits
+presubmits:
+  kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm:
+  - name: pull-cluster-api-bootstrap-provider-kubeadm-verify
+    path_alias: "sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm"
+    always_run: true
+    optional: false
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190626-f0c7a9f-master
+        command:
+        - "./hack/verify-all.sh"
```

Comment:
This file doesn't appear to exist.

Reply:
just merged:
https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/pull/4/files#diff-3e7af3bd412c2b31500218f0692627e8